config.yaml

```diff
@@ -7,15 +7,15 @@ options:
       default: ""
       type: string
       description: "Personal Access Token, required for auto registerting with the given url"
-  quantity:
-      default: 3
-      type: int
-      description: "Quantity of runners to maintain"
   check-interval:
       default: "*/1 * * * *"
       type: string
       description: "Cron interval to check runners, each deployed unit will check on this interval"
-  virt-type:
-      default: "container"
-      type: string
-      description: "Virtualization type for runners, either container or virtual-machine"
+  containers:
```

Comment:
changing this name from `quantity` to `containers` will impact already existing deployments.  Can you deprecate `quantity` but leave it in there and support it?

Reply:
Is there a recommended way of deprecating config values, or is it acceptable to add a prefix to the description and code adequate logic in the code? 
I suppose that virt-type should be taken into account as well, correct? It would make sense I guess.

Reply:
Usually, add a description to the existing config like:

```yaml
    description: |
       DEPRECATED:  use `containers` instead
       <existing description>.  
```

then, make `containers` default to something like `None` or some such field where you inherit the config value of `quantity`.   I've seen this as a general rule, not sure if it's best practice.